### ISSUE TITLE
[UWP][Accessibility] Keyboard behavior in expanded choice sets

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#4424]
# Platform

* UWP

# Details

From Inclusive and Accessible Design Spec Review Report:

### Using Input.ChoiceSet.Expanded.Label.json:

 - Keyboard accessibility, Tab and shift Tab key should be used to move between group of radio button not with in group of radio button. 
 - When using tab key or shift tab key, focus should go to the selected radio button irrespective of its position in group.

See guidelines and example from  WCAG’s Wai-aria from https://www.w3.org/TR/wai-aria-practices/examples/radio/radio-1/radio-1.html

### Using Input.ChoiceSet.Expanded.MultiSelect.Label.json

 - Keyboard accessibility: Up and down arrow keys should move focus within group of checkbox.

See example and guideline from WCAG W3C link https://www.w3.org/TR/wai-aria-practices/examples/checkbox/checkbox-1/checkbox-1.html

**Comment by paulcam206** [comment]
I don't think that WCAG rules apply in UWP. We should be following platform accessibility guidelines, which should already be implemented in XAML.
